modules/exploits/windows/local/linqpad_deserialization_persistence.rb

```diff
@@ -0,0 +1,86 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = NormalRanking # https://docs.metasploit.com/docs/using-metasploit/intermediate/exploit-ranking.html
+
+  # includes file?, directory?
+  include Msf::Post::File
+
+  # includes generate
+  include Msf::Util::DotNetDeserialization
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        # The Name should be just like the line of a Git commit - software name,
+        # vuln type, class. Preferably apply
+        # some search optimization so people can actually find the module.
+        # We encourage consistency between module name and file name.
+        'Name' => 'LINQPad Deserialization Exploit',
+        'Description' => %q{
+          This module exploits a bug in LIQPad up to version 5.48.00. The bug is only exploitable in paid version of software. The core of a bug is cache file containing deserialized data, which attacker can overwrite with malicious payload. The data gets deserialized every time the app restarts.
+        },
+        'License' => MSF_LICENSE,
+        'Author' => [
+          'msutovsky-r7 <martin_sutovsky@rapid7.com>',
+          'James Williams' # original research
+        ],
+        'Platform' => 'win',
+        'SessionTypes' => [ 'shell', 'meterpreter' ],
+        'Targets' => [[ 'Windows', { 'Arch' => ARCH_CMD } ]],
+        # 'Privileged' => true,
+        'References' => [
+          [ 'URL', 'https://trustedsec.com/blog/discovering-a-deserialization-vulnerability-in-linqpad'],
+          [ 'CVE', '1978-1234']
```

Comment:
```suggestion
          [ 'CVE', '2024-53326']
```
Not sure where 1978 came from 😅